examples/meander/meander_example.py

```diff
@@ -0,0 +1,157 @@
+"""
+Meander Test case
+=======================
+
+Solves the test case of a meander.
```

Comment:
Maybe you could be a bit more specific, "meander" is quite a generic term. Idealised 2D sediment and morphodynamics test in a circular channel or something like that?

Reply:
I have replaced the text with "Solves the test case of flow around a 180 degree bend replicating lab experiment 4 in Yen & Lee (1995)."